src/Commands/Concerns/InteractsWithIO.php

```diff
@@ -45,8 +49,21 @@ public function info($string, $verbosity = null)
      */
     public function error($string, $verbosity = null)
     {
-        if (! Str::contains($string, $this->ignoreErrors)) {
-            $this->label($string, $verbosity, 'ERROR', 'red', 'white');
+        $this->label($string, $verbosity, 'ERROR', 'red', 'white');
+    }
+
+    /**
+     * Write a string as raw output.
+     *
+     * @param  string  $string
+     * @return void
+     */
+    public function raw($string)
+    {
+        if (! Str::startsWith($string, $this->ignoreMessages)) {
+            $this->output instanceof OutputStyle
+                ? fwrite(STDERR, $string."\n")
+                : $this->output->writeln($string);
```

Comment:
I'm a bit confused by this piece of code - why does this work? Why check for `OutputStyle` to then not call a method on it? From what I can see, `$this->output` should always be `OutputStyle` within a Laravel `Command` anyway.

The call to `fwrite(STDERR, $str)` is ok for me, but seems like its not quite the Laravel way compared with the rest of the command base. I was fiddling around with it yesterday a bit and there should be some sort of `stderr` support in the Symfony base, but I couldn't get it to work properly. Is this also the reason why you have chosen to call `fwrite` on `stderr` directly?

Reply:
If the condition `$this->output instanceof OutputStyle` passes, this means we are the CLI context, therefore we should/can use the `fwrite` safely. Symfony does support writing for `stderr` using the `$output->getErrorOutput()`, however the `$output` instance we are using at Laravel does not have support for that.

Reply:
Ok got it. Doesn't make much sense to run the `octane:start` command from within php-fpm for example, but I realize its possible. This part seems fine then!